app/views/model_view.py

```diff
@@ -13,57 +14,63 @@ class DonorView(View):
     - PUT: Update and returnn that Donor object
     - DELETE: Delete and return HTTP status code
     '''
+
     def get(self, request):
         try:
             donor = Donor.objects.get(id=request.GET['donor_id'])
             return JsonResponse(donor.serialize(), status=200)
-        except BaseException:
+        except Exception as e:
+            print e.args
             return HttpResponseBadRequest()
 
     def post(self, request):
         try:
             donor = Donor.objects.create(
                 donor_name=request.POST['donor_name'],
                 email=request.POST['email'],
-                want_receipt=request.POST['want_receipt'],
+                want_receipt='want_receipt' in request.POST,
                 telephone_number=request.POST['telephone_number'],
                 mobile_number=request.POST['mobile_number'],
                 address_line=request.POST['address_line'],
                 city=request.POST['city'],
                 province=request.POST['province'],
                 postal_code=request.POST['postal_code'],
                 customer_ref=request.POST['customer_ref'],
-                verified=request.POST['verified']
+                verified='verified' in request.POST
             )
             return JsonResponse(donor.serialize(), status=201)
-        except BaseException:
+        except Exception as e:
+            print e.args
             return HttpResponseBadRequest()
 
     def put(self, request):
         try:
+            request.PUT = QueryDict(request.body)
             donor = Donor.objects.get(id=request.PUT['donor_id'])
-            donor.donor_name = request.PUT['donor_name'],
-            donor.email = request.PUT['email'],
-            donor.want_receipt = request.PUT['want_receipt'],
-            donor.telephone_number = request.PUT['telephone_number'],
-            donor.mobile_number = request.PUT['mobile_number'],
-            donor.address_line = request.PUT['address_line'],
-            donor.city = request.PUT['city'],
-            donor.province = request.PUT['province'],
-            donor.postal_code = request.PUT['postal_code'],
-            donor.customer_ref = request.PUT['customer_ref'],
-            donor.verified = request.PUT['verified']
+            donor.donor_name = request.PUT['donor_name'].split(',')[0]
```

Comment:
I think you should keep the line to just use the request.PUT['donor_name'] and rather change the request itself since we don't expect to have to split donor_name from request when you read this.

Reply:
Where would you like the name string to be parsed?

Reply:
Now handle split at front end.